addToCollection.php

```diff
@@ -0,0 +1,94 @@
+<?php
+?>
+
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+    <title>Add to the Collection</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="normalize.css" type="text/css">
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+
+<header>
+    <img src="images/legoLogo3.png" alt="Harry Potter Lego Collection" />
+</header>
+
+<section>
+    <form method='post' action='index.php'>
+        <div class='formItem'>
+            <label for='item-name'>Lego Set Name</label>
+            <input type='text' name='item-name' id='item-name' pattern='' required />
+        </div>
+
+        <div class='formItem'>
+            <label for='number-of-pieces'>Number of Pieces</label>
+            <input type='number' name='number-of-pieces' id='number-of-pieces' required />
+        </div>
+
+        <div class='formItem'>
+            <label for='age-category'>Age Category</label>
+            <input type='text' name='age-category' id='age-category' pattern='' required />
+        </div>
```

Comment:
All looks good! It's not necessary but I could make a couple of suggestions:
1.  You could provide a min and max value for number-of-pieces. There's nothing stopping someone from typing in a negative value currently.
2. You could turn age-category into a drop box with specified inputs e.g. 8+, 8-14, 16+ etc. This is something I haven't implemented yet myself but it may be worth looking into if you want to make the form more fool proof.